app.py

```diff
@@ -36,8 +43,90 @@ def datasets(index: str):
 @app.route('/datasets/refresh')
```

Comment:
Could you please rename this endpoint to `/refresh`, as we are refreshing multiple things now, not datasets only.